src/Uno.UI/Controls/Window.iOS.cs

```diff
@@ -127,6 +177,13 @@ public override void PressesBegan(NSSet<UIPress> presses, UIPressesEvent evt)
 		/// </summary>
 		public int FocusedViewBringIntoViewOnKeyboardOpensPadding { get; set; }
 
+		private void RaiseKeyEvent(Action<KeyEventArgs> raisePointerEvent, KeyEventArgs args)
+		{
+			_ = GetOwner()?.Dispatcher.RunAsync(
+				CoreDispatcherPriority.High,
+				() => raisePointerEvent(args));
+		}
```

Comment:
```suggestion
		private void RaiseKeyEvent(Action<KeyEventArgs> raiseKeyEvent, KeyEventArgs args)
		{
			_ = GetOwner()?.Dispatcher.RunAsync(
				CoreDispatcherPriority.High,
				() => raiseKeyEvent(args));
		}
```

Also, I'm really worried to raise events in an async way. Is there a specific reason to do it like that?

Reply:
Indeed, this should be not async, otherwise the order could get mixed up

Reply:
It's likely that we were doing this on another platform, but it may not be needed here. Would that be the case here @ajpinedam?

Reply:
Yes @jeromelaban . I followed this pattern I saw for another platform. I will update and make it synchronous.

Reply:
Updated call to syncronous.